ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java

```diff
@@ -1,22 +1,23 @@
 package org.openmrs.reference;
 
+import java.util.List;
+
 import org.junit.Assert;
-import org.junit.Ignore;
+import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.page.AdministrationPage;
 import org.openmrs.reference.page.ModulesPage;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
-import java.util.List;
-
 public class CheckModulesTest extends ReferenceApplicationTestBase {
 
     /**
      * Check the list of modules to be sure they are all started.
      */
-    @Ignore
-    @Category(org.openmrs.reference.groups.BuildTests.class)
+	@Test
+	@Category(BuildTests.class)
```

Comment:
Thanks @insookwa , cc @dkayiwa  @ibacher  could you help us approve the man on this project such that we can see his builds

Reply:
> cc @dkayiwa @ibacher could you help us approve the man on this project such that we can see his builds

@sherrif10 before we call on these `devs` for approval, I would suggest we ensure that the `PR` author has simulated the test locally and its passing.

Reply:
@kdaud  I have tested on both chrome and firefox, both headless false and true